beacon-chain/p2p/broadcaster.go

```diff
@@ -371,7 +368,7 @@ func (s *Service) internalBroadcastDataColumn(
 
 	// Define a one-slot length context timeout.
 	secondsPerSlot := params.BeaconConfig().SecondsPerSlot
-	oneSlot := time.Duration(secondsPerSlot) * time.Second
+	oneSlot := time.Duration(secondsPerSlot) * time.Hour
```

Comment:
why is time changed to Hour?

Reply:
😱 Nice catch. I changed it for debug purposes and forgot to set it back correctly.

Fixed in 4a62bf15d3471285b93623f3eaa01038f9a53994.